lib/rubocop/cop/layout/space_in_lambda_literal.rb

```diff
@@ -24,10 +24,10 @@ class SpaceInLambdaLiteral < Cop
         include RangeHelp
 
         ARROW = '->'.freeze
-        MSG_REQUIRE_SPACE = 'Use a space between `->` and opening brace ' \
-                            'in lambda literals'.freeze
+        MSG_REQUIRE_SPACE = 'Use a space between `->` and opening ' \
+                            'parenthesis in lambda literals'.freeze
```

Comment:
Maybe we can use `{` instead of the word here. Would be even clearer IMO (and in line with the usage of `->` beforehand).

Reply:
Seems reasonable, though I'm guessing you meant `(` rather than `{`?

Reply:
Yeah, sorry. See, words are confusing! 😆